.circleci/config.yml

```diff
@@ -85,11 +115,72 @@ jobs:
           path: frontend/reports/
       - store_artifacts:
           path: frontend/coverage/
+  deploy:
+    executor: docker-executor
+    steps:
+      - attach_workspace:
+          at: .
+      - run:
+          name: Build backend assets
+          command: yarn build
+      - run:
+          name: Build frontend assets
+          command: cd frontend && yarn build
+      - run: curl -v -L -o cf-cli_amd64.deb 'https://cli.run.pivotal.io/stable?release=debian64&source=github'
+      - run: sudo dpkg -i cf-cli_amd64.deb
```

Comment:
Is there anyway we can use a different "executor" or "orb" that has this backed in so we don't have to install it every run (or maybe it gets cached so it isn't a big deal?)

Reply:
That was my original approach, but I couldn't find any documentation for a login command in the orb docs. I'll look again.  Actually, this isn't getting cached, just the node modules are cached unfortunately. This install will only run when the deploy job runs, and the deploy job will only run on branches labeled `main` `staging` or `production` so it isn't happening for every push on every feature branch at least.